src/app/modules/main/app_search/module.nim

```diff
@@ -68,6 +68,9 @@ method getModuleAsVariant*(self: Module): QVariant =
 
 proc getChatSubItems(self: Module, chats: seq[ChatDto]): seq[location_menu_sub_item.SubItem] =
   for chatDto in chats:
+    if chatDto.isHiddenChat:
```

Comment:
Maybe I'm once again confused by the NIM syntax but... this looks like you're calling a property but you only defined a function/proc in the DTO

Reply:
Yeah, in Nim `()` is optional. So it does look like a property. I like it personally. It also saved 2 big characters :joy: